src/sdk/namespace_fs.js

```diff
@@ -415,6 +415,16 @@ const versions_dir_cache = new LRUCache({
     max_usage: config.NSFS_DIR_CACHE_MAX_TOTAL_SIZE,
 });
 
+/**
+ * @typedef {{
+ *  state: 'LOW' | 'OK' | undefined
+ * }} NsfsFsSpaceStateCache
+ * @type {NsfsFsSpaceStateCache}
+ */
+const nsfs_fs_space_cache = {
+    state: undefined,
+};
```

Comment:
this is global for all buckets but different buckets could be on different filesystems so we might need a map where the key is devid or fsid ...